app/src/main/res/values/strings.xml

```diff
@@ -105,4 +105,7 @@
     <string name="description_third_image">Third image</string>
     <string name="description_fourth_image">Fourth image</string>
     <string name="description_first_image">First image</string>
+    <string name="title_post_detail">%1$s Post</string>
+    <string name="title_selected_image_deital">Image detail</string>
+>>>>>>> fix-image-detail
```

Comment:
ERROR: [MergeMarker] Missing merge marker?
 Many version control systems leave unmerged files with markers such as <<< in the source code. This check looks for these markers, which are sometimes accidentally left in, particularly in resource files where they don't break compilation.